registry/default/blocks/nsfw-warning.tsx

```diff
@@ -24,7 +27,7 @@ const NsfwWarning = ({
         <Typography variant="sm">
           The post author marked this post as NSFW
```

Comment:
the texts are still hard coded like we discussed this should be provided via a prop